src/renderer/js/component/shapeShift/internal/active-shift.jsx

```diff
@@ -0,0 +1,147 @@
+// @flow
+import * as React from "react";
+import QRCode from "qrcode.react";
+import * as statuses from "constants/shape_shift";
+import Address from "component/address";
+import Link from "component/link";
+import type { Dispatch } from "redux/actions/shape_shift";
+
+type Props = {
+  shiftState: ?string,
+  shiftCoinType: ?string,
+  shiftDepositAddress: ?string,
+  shiftReturnAddress: ?string,
+  shiftOrderId: ?string,
+  originCoinDepositMax: ?number,
+  clearShapeShift: Dispatch,
+  doShowSnackBar: Dispatch,
+  getActiveShift: Dispatch,
+};
+
+class ActiveShapeShift extends React.PureComponent<Props> {
+  constructor() {
+    super();
+    // $FlowFixMe
+    this.continousFetch = undefined;
+  }
+
+  componentDidMount() {
+    const { getActiveShift, shiftDepositAddress } = this.props;
+
+    getActiveShift(shiftDepositAddress);
+    // $FlowFixMe
+    this.continousFetch = setInterval(() => {
+      getActiveShift(shiftDepositAddress);
+    }, 10000);
+  }
+
+  componentWillUpdate(nextProps: Props) {
+    const { shiftState } = nextProps;
+    if (shiftState === statuses.COMPLETE) {
+      this.clearContinuousFetch();
+    }
+  }
+
+  componentWillUnmount() {
+    this.clearContinuousFetch();
+  }
+
+  clearContinuousFetch() {
+    /// $FlowFixMe
+    clearInterval(this.continousFetch);
+    // $FlowFixMe
+    this.continousFetch = null;
+  }
+
+  render() {
+    const {
+      shiftCoinType,
+      shiftDepositAddress,
+      shiftReturnAddress,
+      shiftOrderId,
+      shiftState,
+      originCoinDepositMax,
+      clearShapeShift,
+      doShowSnackBar,
+    } = this.props;
+
+    return (
+      <div>
+        {shiftState === statuses.NO_DEPOSITS && (
+          <div>
+            <p>
+              Send up to{" "}
+              <span className="credit-amount--bold">
```

Comment:
This whole element should be the same style IMO. I'd be fine with just `credit-amount--bold` for the whole thing.